straxen/plugins/peaklet_processing.py

```diff
@@ -352,7 +364,7 @@ def _peak_saturation_correction_inner(channel_saturated, records, p,
         if (b_sumwf[ref] > 1).sum() < min_reference_length:
             # the same condition applies to the waveform model
             continue
-        if np.sum(b[ref]) * to_pe[ch] / np.sum(b_sumwf[ref]) < 1:
+        if np.sum(b[ref]) * to_pe[ch] / np.sum(b_sumwf[ref]) > 1:
```

Comment:
Was this simply a bug and just corrected now?

Reply:
I think this really is a bug